src/main/webapp/assets/css/contactStyle.css

```diff
@@ -0,0 +1,102 @@
+/*
+ * Containers
+ */
+
+#col-container {
+  width: 80%;
+  margin: 0 10% 50px 10%;
+}
+
+#page-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+#map-container {
+  flex: 1 1 400px;
+  padding: 0 15px 0 15px;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+}
+
+.col {
+  display: flex;
+  flex-direction: column;
+  padding-left: 1rem;
+}
+
+/*
+ * Map (will disappear when screen is too small)
+ */
+
+#map {
+  background: var(--grey);
+  height: 100%;
+  min-width: 100px;
+  width: 100%;
+}
+
+#title {
+  padding: 0 0 0 15px;
+}
+
+/*
+ * Mate
+ */
+
+#podmates {
+  padding: 15px;
+  background-color: #a98585;
+  display: flex;
+  justify-content: space-evenly;
+  flex-direction: column;
+  flex: 1 1 400px;
+}
+
+@media screen and (max-width: 1077px) {
+  #podmates {
+    height: auto;
+    margin: 0 16px 0 16px;
+    padding: 0 0 16px 0;
+  }
+
+  #col-container {
+    margin: 0 0 50px 0;
+  }
+}
+
+.mate {
+  background: #fff;
+  border-radius: 15px;
+  padding: 16px;
+  margin: 16px;
+}
+
+img {
+  width: auto;
+  max-height: 50px;
+  border-radius: 50%;
+}
+
+.svg {
+  padding-left: 10px;
+}
+
+h4 {
+  margin-block-start: 0;
+  margin-block-end: 0.33em;
+}
+
+a {
+  text-decoration: none;
+  color: var(--light-maroon);
+  margin-right: 5px;
```

Comment:
ordering :0
also, are the links a different color than the in the PR's screenshots? Cause I think that's a great idea, the blue/purple vibe doesn't really go with our aesthetic :)

Reply:
Will change the PR's screenshots!!